OSDOCS-5442: add info on lvmd and ovn yaml files (#57324)
openshift/openshift-docs: modules/microshift-lvmd-yaml-creating.adoc

// Module included in the following assemblies:
//
// * microshift_storage/microshift-storage-plugin-overview.adoc

:_content-type: PROCEDURE
[id="microshift-lvmd-yaml-creating_{context}"]
= Creating an LVMS configuration file

> When {product-title} runs, it uses LVMS configuration from `/etc/microshift/lvmd.yaml`, if provided. You must place any configuration files you make into the `/etc/microshift/` directory.

Reviewer: ```suggestion
When {product-title} runs, it uses LVMS configuration from `/etc/microshift/lvmd.yaml`, if provided. You must place any configuration files that you create into the `/etc/microshift/` directory.
```

// Module included in the following assemblies:
//
// * microshift_storage/microshift-storage-plugin-overview.adoc

:_content-type: PROCEDURE
[id="microshift-lvmd-yaml-creating_{context}"]
= Creating an LVMS configuration file

When {product-title} runs, it uses LVMS configuration from `/etc/microshift/lvmd.yaml`, if provided. You must place any configuration files you make into the `etc/microshift/` directory.

.Procedure

* To create the `lvmd.yaml` configuration file, run the following command:

[source, terminal]
----
$ sudo cp /etc/microshift/lvmd.yaml.default /etc/microshift/lvmd.yaml
----

> <2> Volume group. You can set a custom volume group name here.

Reviewer 2: This looks like a reference to something in the old example. I think we can delete the line?

```suggestion
```

> //Q: What's the instruction for multi-vg?

Reviewer 2: Most users will not have multiple volume groups and storage classes, so let's treat that as a separate procedure and document it later.

// Module included in the following assemblies:
//
// * microshift_storage/microshift-storage-plugin-overview.adoc

:_content-type: PROCEDURE
[id="microshift-lvmd-yaml-creating_{context}"]
= Creating an LVMS configuration file

When {product-title} runs, it uses LVMS configuration from `/etc/microshift/lvmd.yaml`, if provided. You must place any configuration files you make into the `etc/microshift/` directory.

.Procedure

* To create the `lvmd.yaml` configuration file, run the following command:

[source, terminal]
----

> sudo cat >/etc/microshift/lvmd.yaml <<EOF
> device-classes:
> - default: true
>   name: default
>   spare-gb: 0
>   volume-group: rhel <1>
> socket-name: /run/lvmd/lvmd.socket
> EOF

Reviewer 3: ```suggestion
We have a default file - let's just copy it
```suggestion
$ sudo cp /etc/microshift/lvmd.yaml.default /etc/microshift/lvmd.yaml
```
```
